pom.xml

```diff
@@ -214,9 +214,9 @@
         <version>${slf4j.version}</version>
       </dependency>
       <dependency>
-        <groupId>log4j</groupId>
-        <artifactId>log4j</artifactId>
-        <version>${log4j.version}</version>
+        <groupId>ch.qos.reload4j</groupId>
```

Comment:
Do not remove the dependencyManagement section for log4j, spark-hbase still depends on it.

Also, this doesn't replace log4j with reload4j, this just adds reloadj4, please fix the description. (after adding back the log4 dependency management entry)

Also, keep the ticket description and commit message in sync.

The spark connector tests have both the log4 and reload4j backends on the classpath, it's a mess, and needs to be fixed (but not in this ticket.)